commit b2841bbade80dd0acf52ed633e67e73e6f663f4d
Author: q66 <q66@chimera-linux.org>
Date:   Fri May 12 02:56:09 2023 +0200

    disable defcheck
    
    this fails on false positive junk on aarch64

diff --git a/libgimp/meson.build b/libgimp/meson.build
index 82a91e4..ab1eac1 100644
--- a/libgimp/meson.build
+++ b/libgimp/meson.build
@@ -571,7 +571,7 @@
   endif
 endif
 
-if not platform_windows and not platform_osx
+if false and not platform_windows and not platform_osx
   # Verify .def files for Windows.
   # Ironically we only check this on non-Windows platform, since the
   # script expects .so libraries, and I'm not sure that the `nm` tool is
